journal-writer/src/main/scala/akka/stream/alpakka/persistence/writer/package.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.persistence
+
+import akka.actor.{ ActorContext, ActorRef }
+import akka.persistence.query.{ EventEnvelope, EventEnvelope2 }
+import akka.persistence.{ AtomicWrite, PersistentRepr }
+
+import scala.collection.immutable.{ Map, Seq }
+import scala.reflect.ClassTag
+
+package object writer {
+  val unsupported: Throwable = new IllegalArgumentException("JournalWriter only accepts EventEnvelope, immutable.Seq[EventEnvelope], EventEnvelope2 and immutable.Seq[EventEnvelope2]")
+
+  def replyWithFailure(replyTo: ActorRef, cause: Throwable = unsupported)(implicit ctx: ActorContext, self: ActorRef): Unit = {
+    replyTo ! akka.actor.Status.Failure(cause)
+    ctx.stop(self)
+  }
+
+  def replyWithSuccess(replyTo: ActorRef, msg: String = "ack")(implicit ctx: ActorContext, self: ActorRef): Unit = {
```

Comment:
I agree with @patriknw here. I understand each person have a different style that he likes but I would like to see these refactored simply because it is of a different style than is used elsewhere in this project. We are much more lenient in Alpakka than in Akka because it is more community driven but I think this is just too different from other subprojects here.